Makefile

```diff
@@ -245,6 +245,12 @@ shellcheck:
 	@find tee-worker/scripts/litentry/ -name "*.sh" | xargs shellcheck -S warning
 	@echo "Ok"
 
+# generate release tag
+
+.PHONY: generate-tag ## generate full release tag
+generate-tag:
+	@./scripts/generate-release-tag.sh
```

Comment:
did you forget to `git add` some file? Or it should be `generate-release-notes`?

Reply:
Good catch - there was a script to generate that composited tag `pxxx-ixxx-bxxx` 😂  - but not valid anyone after we decided to use a general tag

I'll remove it